lib/Dialect/XTenNN/Interfaces/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+#
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# (c) Copyright 2022 Advanced Micro Devices, Inc.
+
+add_mlir_dialect_library(XTenNNInterfaces
```

Comment:
In this file, the formatting is not consistent with the other `CMakeLists.txt` files.

Reply:
This is fixed, thanks